tests/layer_tests.py

```diff
@@ -0,0 +1,65 @@
+import pytest
+import openvino as ov
+core = ov.Core()
+
+import os, sys
+core.add_extension('/Users/pavel-esir/devel/openvino_tokenizers/build-Debug/src/libopenvino_tokenizersd.dylib')
+os.environ.update({"OV_TOKENIZER_PREBUILD_EXTENSION_PATH": "/Users/pavel-esir/devel/openvino_tokenizers/build-Debug/src/libopenvino_tokenizersd.dylib"})
+sys.path.append("/Users/pavel-esir/devel/openvino_tokenizers/python")
```

Comment:
`import openvino_tokenizers` or `from openvino_tokenizers import _get_factory`

Reply:
ohh, sory. I forgot to delete this in the last moment

Reply:
removed those lines